src/libasr/pass/intrinsic_subroutines.h

```diff
@@ -21,6 +21,7 @@ the code size.
 enum class IntrinsicImpureSubroutines : int64_t {
     RandomNumber,
     RandomInit,
+    Get_Command,
```

Comment:
This is the convention that we use for intrinsic function names:
```suggestion
    GetCommand,
```